src/components/Dashboard/FirmsAddressList.vue

```diff
@@ -53,35 +53,41 @@
                       <span>{{ getCountryName(businessAddress.deliveryAddress.addressCountry) }}</span>
                     </li>
                   </ul>
+                  <ul class="address-subtitle pre-line" v-else>
+                    <li class="address-line1">Not Entered</li>
```

Comment:
I recommend a new classname here. AFAIK these classnames are only used for selecting elements during testing.

It looks like your change broke a unit test, that you need to fix. Also it would be nice to add some test cases that verify your change versus normal behaviour.